models/experimental/functional_roberta/tests/test_ttnn_optimized_roberta.py

```diff
@@ -0,0 +1,74 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import ttnn
+import torch
+import pytest
+import tt_lib
+import transformers
+
+from models.demos.bert.tt import ttnn_optimized_bert, ttnn_bert
+from ttnn.model_preprocessing import preprocess_model_parameters
+from tests.ttnn.utils_for_testing import assert_with_pcc
+from models.utility_functions import skip_for_wormhole_b0
+
+from transformers import RobertaForQuestionAnswering, RobertaConfig
+
+
+@skip_for_wormhole_b0()
+@pytest.mark.models_device_performance_bare_metal
+@pytest.mark.models_performance_bare_metal
```

Comment:
What is this? Is this a functional test, e2e perf, device perf? Why are you copy pasting decorators like this?

Reply:
This is a functional test. 
The decorators were added unintentionally.
Apologies for any confusion. 

Thank you.